spark/v3.3/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestRewriteDataFilesProcedure.java

```diff
@@ -580,8 +614,11 @@ public void testDefaultSortOrder() {
 
     assertEquals(
         "Action should rewrite 2 data files and add 1 data files",
-        ImmutableList.of(row(2, 1)),
-        output);
+        row(2, 1),
+        Arrays.copyOf(output.get(0), 2));
```

Comment:
Now that I see this, I think we should really just use the real amount that we get programmatically

Reply:
I've updated checks to compare results with programmatically determined file sizes